examples/proxy/psp22_metadata_upgradeable/README.md

```diff
@@ -0,0 +1,5 @@
+## PSP22 contract (ERC20 analogue) with 'Metadata' extension
+
+Implementation of 'Metadata' extension for [PSP-22](https://github.com/w3f/PSPs/blob/master/PSPs/psp-22.md) token standard in Polkadot blockchain, which allows to add Metadata to the PSP-22 token.
```

Comment:
Could you add more description here that it is part of upgradable example and you want to show something=)